tuf/ngclient/_internal/requests_fetcher.py

```diff
@@ -47,6 +48,11 @@ def __init__(self):
         # Some cookies may not be HTTP-safe.
         self._sessions = {}
 
+        # Default settings
+        self.sleep_before_round: int = 4  # seconds
```

Comment:
er, I think you mixed up sleep_before_round and socket_timeout: this is going to make life painful

Reply:
oh 🤦